test/GSNProvider.test.js

```diff
@@ -518,6 +519,59 @@ describe('GSNProvider', function () {
     });
 
   });
+
+  context('detecting relays added', function () {
+    before('getting relayHub', async function () {
+      this.relayHub = await getRelayHub(this.web3);
+    });
+
+    beforeEach('creating provider', async function () {
+      this.gsnProvider = new GSNProvider(PROVIDER_URL, HARDCODED_RELAYER_OPTS);
+      this.greeter.setProvider(this.gsnProvider);
+      await this.greeter.methods.greet("Hello").send({ from: this.sender, useGSN: true });
```

Comment:
Why is this line needed?

Reply:
It's needed because `RelayClient`

Reply:
Good point. Let's leave it like this for now.